packages/amplify-codegen/src/amplify-plugin-index.js

```diff
@@ -1,12 +1,20 @@
 const path = require('path');
 
 const pluginName = 'codegen';
+const codeModule = '@aws-amplify/amplify-codegen-core';
 
 async function executeAmplifyCommand(context) {
   let commandPath = path.normalize(path.join(__dirname, '../commands'));
   commandPath = path.join(commandPath, pluginName, context.input.command);
-  const commandModule = require(commandPath);
-  await commandModule.run(context);
+
+  // Call types plugin:
+  if (context.input.command == 'types' && context.input.options != undefined && context.input.options['plugin'] == true) {
```

Comment:
If the third argument in "args" of launch.json is "-plugin", then the Typescript types generator plugin (appsync-typegen-plugin package) will generates the types, instead of the regular flow of graphql-types-generator package generating the types.